_sass/addon/variables.scss

```diff
@@ -28,3 +28,14 @@ $bottom-min-height: 35rem !default;
 /* syntax highlight */
 
 $code-font-size: 0.85rem !default;
+
+/* fonts */
+
+$body-font: 'Source Sans Pro', 'Microsoft Yahei', sans-serif;
+$heading-font: Lato, 'Microsoft Yahei', sans-serif;
+$h1-size: 1.9rem;
```

Comment:
1. Rename the variables for font-family, what's more
2. Drop the definition of heading font sizes, as they will be produced by a formula in the future. 
3. Finally, please take care of the CI results and make improvements, thanks!